example_package/Cargo.toml

```diff
@@ -6,6 +6,8 @@ edition = "2021"
 [dependencies]
 # The code generated by roslibrust_codegen has dependendencies
 # We need to depend on the crate at build time so that the generate code has access to these dependencies
+roslibrust_common = { path = "../roslibrust_common" }
+# THIS SHOULDN'T BE NEEDED, BUT IS FOR NOW GOTTA FIX LEAKING DEPENDENCIES
```

Comment:
Can you make a tracking issue for this?

Reply:
https://github.com/RosLibRust/roslibrust/issues/216